mindsdb/integrations/handlers/google_content_shopping_handler/__init__.py

```diff
@@ -0,0 +1,20 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __version__ as version, __description__ as description
+
+try:
+    from .google_content_shopping_handler import GoogleContentShoppingHandler as Handler
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = 'Google Content Shopping'
+name = 'google_content_shopping'
+type = HANDLER_TYPE.DATA
+icon_path = 'icon.png'
+
+__all__ = [
+    'Handler', 'version', 'name', 'type', 'title', 'description',
+    'import_error', 'icon_path'
```

Comment:
If possible, please add also `connection_args` and `connection_args_example`

Reply:
Done.